### PR TITLE
Enabled server-side Secure ETCD and Secure GRPC 

### PR DESCRIPTION
**Pull request by seswarrajan**

### CODE REVIEW

File: src/service/server/server.go [hunk 1]
```diff
@@ -44,8 +46,13 @@ func (s *Server) InitServer() {
 		kg.Printf("Successfully KVMServer Listening on port %s", s.port)
 	}
 
+	creds, err := credentials.NewServerTLSFromFile(ct.ServerCertPath, ct.ServerKeyPath)
+	if err != nil {
+		kg.Errf("Failed to setup TLS: %v", err)
```
Comment (nyrahul) [inline src/service/server/server.go:51]: return needed here.

File: src/service/server/server.go [hunk 2]
```diff
@@ -44,8 +46,13 @@ func (s *Server) InitServer() {
...
+	}
+
 	// Start gRPC server and register for protobuf
-	gRPCServer := grpc.NewServer()
+	gRPCServer := grpc.NewServer(grpc.Creds(creds))
 	if gRPCServer == nil {
 		kg.Err("Failed to serve gRPCServer is null")
```
Comment (nyrahul) [inline src/service/server/server.go:57]: Return needed here.

**Review by nyrahul — CHANGES_REQUESTED** [review]
Thanks, please check the comments.

**Comment by seswarrajan** [comment]
Closing this PR as this is a duplicate of https://github.com/kubearmor/kvm-service/pull/30

https://github.com/kubearmor/kvm-service/pull/30 is the updated PR

**Pull request closed without merging**